### PR TITLE
Filtering localization

### PR DESCRIPTION
**Pull request by aekeus**
Initialized localization strings in session.setPermissionRequestHandler

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -366,8 +368,8 @@ app.on('ready', () => {
       initiateSessionStateSave()
     })
 
-    Extensions.init()
     Filtering.init()
+    Extensions.init()
```
Comment (bbondy) [inline app/index.js]: Does this need to be re-ordered?

File: app/filtering.js [hunk 2]
```diff
@@ -235,31 +236,31 @@ function registerForHeadersReceived (session) {
...
-    fullscreen: {
-      action: 'use fullscreen mode'
-    },
-    openExternal: {
-      action: 'open an external application'
-    }
-  }
+  let permissions = {}
   session.setPermissionRequestHandler((webContents, permission, cb) => {
+    permissions = {
```
Comment (bbondy) [inline app/filtering.js]: prefer to only set this once if not already set, because it will be a callback called a lot.

**Comment by bbondy** [comment]
If you remove the uneeded reordering and then change to only set the strings once in the permission callback then please go ahead and merge. Thanks!

File: app/index.js (hunk 1 above)
Comment (aekeus) [inline app/index.js]: Nope, mistake. Will fix.

File: app/filtering.js (hunk 2 above)
Comment (aekeus) [inline app/filtering.js]: Ok, will cache after first call.

**Pull request merged**